src/main/java/net/pms/dlna/DLNAImageProfile.java

```diff
@@ -934,9 +934,9 @@ protected static void checkPNG(
 
 		PNGInfo pngInfo = (PNGInfo) imageInfo;
 		if (pngInfo.getColorType() != null) {
-			switch (pngInfo.getColorType()) {
-				case Greyscale:
-				case GreyscaleWithAlpha:
+			switch (pngInfo.getColorType().getNumericValue()) {
+				case 0:
+				case 4:
```

Comment:
```suggestion
				case 0: // Greyscale
				case 4: // GreyscaleWithAlpha
```

Reply:
Even better would be to reference the static instances instead of using integer values. All the previous enum values are now static instances, so all that's actually needed is to change from camel cast (which shouldn't be used for enum's either) to "upper case with underscores".